src/seedu/addressbook/parser/Parser.java

```diff
@@ -54,6 +40,13 @@
                     + " et/(?<examEndTime>[^/]+)"
                     + " dt/(?<examDetails>[^/]+)");
 
+    public static final Pattern ATTENDANCE_ARGS_FORMAT = // '/' forward slashes are reserved for delimiter prefixes
+            Pattern.compile("n/(?<indexOfPerson>[^/]+)"
+                          + " att/(?<isPresent>[^/]+)");
+
+    public static final Pattern VIEW_ATTENDANCE_ARGS_FORMAT = // '/' forward slashes are reserved for delimiter prefixes
```

Comment:
just use the PERSONS_INDEX_ARG_FORMAT and save the user from typing "n/" as well